tests/System.IO.Pipelines.Tests/SocketsFacts.cs

```diff
@@ -109,41 +109,41 @@ public void CanCreateWorkingEchoServer_PipelineSocketServer_NonPipelineClient()
             Assert.Equal(MessageToSend, reply);
         }
 
-        [Fact(Skip = "System.TypeLoadException : A value type containing a by-ref instance field, such as Span<T>, cannot be used as the type for a class instance field.")]
-        public async Task RunStressPingPongTest_Libuv()
+        [Fact]
+        public void RunStressPingPongTest_Libuv()
         {
             var endpoint = new IPEndPoint(IPAddress.Loopback, 5040);
 
             using (var thread = new UvThread())
             using (var server = new UvTcpListener(thread, endpoint))
             {
                 server.OnConnection(PongServer);
-                await server.StartAsync();
+                server.StartAsync().Wait();
 
                 const int SendCount = 500, ClientCount = 5;
                 for (int loop = 0; loop < ClientCount; loop++)
                 {
-                    using (var connection = await new UvTcpClient(thread, endpoint).ConnectAsync())
+                    using (var connection = new UvTcpClient(thread, endpoint).ConnectAsync().GetResultSync())
                     {
                         try
                         {
-                            var tuple = await PingClient(connection, SendCount);
+                            var tuple = PingClient(connection, SendCount).GetResultSync();
```

Comment:
Instead of doing these changes, does it make sense to just change PingClient and keep the test as async Task?
https://github.com/ahsonkhan/corefxlab/blob/FixingTests/tests/System.IO.Pipelines.Tests/SocketsFacts.cs#L185